src/applications/ivc-champva/10-10D/pages/ApplicantDependentStatus.jsx

```diff
@@ -39,36 +32,26 @@ function generateOptions({ data, pagePerItemIndex }) {
 
   return {
     options,
-    useFirstPerson,
-    relativePossessive,
     relativeBeingVerb,
-    applicant,
-    personTitle,
-    keyname: KEYNAME,
-    primary: PRIMARY,
-    secondary: SECONDARY,
-    currentListItem,
+    ...bp,
+    ...PROPERTY_NAMES,
```

Comment:
Condensed down into these two spreads since we don't need to directly deal with most of these properties in this function.